Hazel/src/Hazel/Debug/Instrumentor.h

```diff
@@ -137,7 +144,9 @@ namespace Hazel {
 		~InstrumentationTimer()
 		{
 			if (!m_Stopped)
+			{
 				Stop();
+			}
```

Comment:
Why did you add curly brackets here? They're redundant

Reply:
To comply to the coding conventions.

Reply:
There are other places inside the code that has curly brackets on one-line if statements, too.
Look at lines 185 to 195 - that is a one-line statement, too .. should remove the curly brackets there, too?
Or look at lines 64 to 67 - there are curly brackets, too (in a one-line statement.

Reply:
As long as that's the code style that this project follows, I guess we should keep it consistent